### PR TITLE
[R4R]generate mainnet and testnet

### PR DESCRIPTION
**Pull request by guagualvcha**
### Description
Add auto generate genesis file for testnet and mainnet.
### Preflight checks

- [ ] build passed (`make build`)
- [ ] tests passed (`make test`)
- [ ] manual transaction test passed

### Already reviewed by

...

### Related issues

... reference related issue #'s here ...

### CODE REVIEW

File: generate-genesis.js [hunk 1]
```diff
@@ -7,6 +7,22 @@ const web3 = require("web3")
 const validators = require("./validators")
 const init_holders = require("./init_holders")
 
+
+program.option("--bscChainId <bscChainId>",
+    "bscChainId",
+    "0060");
+program.option("-c, --chainid <chainid>", "chain id", "714")
```
Comment (HaoyangLiu) [inline generate-genesis.js:14]: Do we need to specify this parameter in generating command?

Comment (guagualvcha) [reply generate-genesis.js:14]: No need

Comment (guagualvcha) [reply generate-genesis.js:14]: This option maybe useless, let me double check

Comment (guagualvcha) [reply generate-genesis.js:14]: ok, this affect:
image
do not affect the bytes code. I will fix this as well

**Review by HaoyangLiu — APPROVED** [review]

**Review by yutianwu — APPROVED** [review]

**Pull request merged**